audioread/ffdec.py

```diff
@@ -264,12 +264,14 @@ def close(self):
             # ffmpeg closes normally on its own, but never updates
             # `returncode`.
             self.proc.poll()
+            self.proc.stdout.close()
+            self.proc.stderr.close()
```

Comment:
The comment above doesn't apply to these lines, so it would be nice to use a blank line above them—and to write a new comment explaining why these are here. (For posterity, for example, we might want to record why this goes *above* the `if` block that happens next.)